content/webapp/pages/guides/exhibitions/[id]/[type]/[stop]/index.tsx

```diff
@@ -119,8 +121,35 @@ const ExhibitionGuidePage: FunctionComponent<Props> = props => {
     stopNumber,
     totalStops,
   } = props;
+  const headerRef = useRef(null);
+
+  useEffect(() => {
+    // We measure the height of the Header element with a ResizeObserver and
+    // update the sticky top position of the StickyPlayer element any time it
+    // changes
+    const resizeObserver = new ResizeObserver(([entry]) => {
+      document.documentElement.style.setProperty(
+        '--stop-header-height',
+        `${entry.contentRect.height}px`
+      );
+    });
+
+    if (headerRef.current) {
+      resizeObserver.observe(headerRef.current);
+    }
+
+    return () => {
+      resizeObserver.disconnect();
+    };
+  }, [headerRef.current]);
```

Comment:
I had to double-check this as I think I'd googled it before for myself. I believe we're not meant to use refs in dependency arrays like this. Does it fire as expected?

If not, looks like the workaround would be to create the ref using `useCallback` instead of `useRef`:
https://medium.com/@teh_builder/ref-objects-inside-useeffect-hooks-eb7c15198780